modules/external-roles/README.md

```diff
@@ -18,6 +18,20 @@ service-account2@project_id.iam.gserviceaccount.com:
 
 See [example](../../examples/external-roles) of using the module
 
+If `project_id` variable provided then only service name may be passed to map instead of whole service account:
+
+```
+service1:
```

Comment:
is this related to our service specified in the project.yaml?

Reply:
It's not related to our services, it means only left part of service account can be speficied

Reply:
okay, but shouldn't we add like this: (in the README service is a bit confusing)
```
roles_map = {
  service_account_1 = {
    project_id_1 = [
        role1,
        role2
],
     project_id_2 = [
        role1,
        role2
],
}
```